cache/memcached_client.go

```diff
@@ -305,12 +305,22 @@ func (c *MemcachedClient) Stop() {
 	c.client.Close()
 }
 
-func (c *MemcachedClient) SetAsync(key string, value []byte, ttl time.Duration) error {
-	return c.setAsync(key, value, ttl, func(key string, buf []byte, ttl time.Duration) error {
+func (c *MemcachedClient) SetMultiAsync(data map[string][]byte, ttl time.Duration) {
+	c.setMultiAsync(data, ttl, func(key string, buf []byte, ttl time.Duration) error {
 		return c.client.Set(&memcache.Item{
 			Key:        key,
-			Value:      value,
```

Comment:
The old method was actually using the wrong var here, setting `value` from the argument to `SetAsync` instead of `buf` being passed to the closure.